src/Integration.Vsix.UnitTests/Integration.Vsix.UnitTests.csproj

```diff
@@ -70,6 +69,14 @@
     
   </ItemGroup>
 
+  <!-- Only need to reference the analyzer common assembly for testing. -->
+  <Import Project="..\EmbeddedSonarAnalyzer.props" />
+  <ItemGroup Label="Embedded SonarLint analyzers">
+    <!-- Reference the packages we want to import, but we don't want to include them as analyzers for project -->
+    <PackageReference Include="SonarAnalyzer.CSharp" Version="$(EmbeddedSonarAnalyzerVersion)" ExcludeAssets="all" PrivateAssets="all" />
+    <Reference Include="$(NuGetPackageRoot)SonarAnalyzer.CSharp\$(EmbeddedSonarAnalyzerVersion)\analyzers\SonarAnalyzer.dll" />
+  </ItemGroup>
```

Comment:
The tests don't need all of the extra properties/config that the product code does; it just needs to reference the one assembly it tests.